lib/jsdom/browser/Window.js

```diff
@@ -34,8 +34,17 @@ const jsGlobals = require("./js-globals.json");
 const GlobalEventHandlersImpl = require("../living/nodes/GlobalEventHandlers-impl").implementation;
 const WindowEventHandlersImpl = require("../living/nodes/WindowEventHandlers-impl").implementation;
 
-exports.createWindow = function (options) {
-  return new Window(options);
+let beforeParse;
+const beforeParseDefaultOptions = { isMain: false };
+
+exports.setBeforeParse = function (cb) {
```

Comment:
beforeParse should not be a module-level variable, and `setBeforeParse` should not be needed. It should be one of the window options, not separate.